source/common/upstream/cds_api_impl.cc

```diff
@@ -70,24 +58,22 @@ void CdsApiImpl::parseResponse(const Http::Message& response) {
     }
   }
 
-  stats_.update_success_.inc();
+  runInitializeCallbackIfAny();
+}
+
+void CdsApiImpl::onConfigUpdateFailed(const EnvoyException* e) {
+  UNREFERENCED_PARAMETER(e);
```

Comment:
nit: most code in this case just deletes e in the signature and then you don't need the macro.